pcap-int.h

```diff
@@ -409,6 +409,11 @@ int	pcap_offline_read(pcap_t *, int, pcap_handler, u_char *);
  * #defining them to be snprintf or vsnprintf, respectively, or by
  * defining our own versions and exporting them.
  */
+
```

Comment:
The only time `snprintf()` will be referred to is if `HAVE_SNPRINTF` is defined.

If `HAVE_SNPRINTF` is defined for versions of MSVC that don't provide `snprintf()`, that's a bug in whatever's defining `HAVE_SNPRINTF`, and the way to fix it is not to define `HAVE_SNPRINTF`.

I.e., never ever ever ever ever ever define `HAVE_SNPRINTF` in an MSVC project file unless the project file somehow checks the version of MSVC and defines it only in versions of MSVC that have it.  If an MSVC project file can't do that, then you need to do something such as

    #if defined(_MSC_VER)
        #if _MSC_VER >= 1900
            #define HAVE_SNPRINTF
        #endif
    #endif

(don't assume that, on Windows, this is being compiled by MSVC, it could be compiled by MinGW's GCC).

Note, however, that `pcap-stdinc.h` is already doing checks of this sort, so that might be the place to add that.